assets/css/style.css

```diff
@@ -542,113 +585,1066 @@ button:disabled {
-    display: none;
+  .navbar {
+    display: block;
+    width: 100%;
+    height: 60px;
+    background-color: #3c3a39;
   }
 
-  .about-section {
-    display: none;
+  .nav-flex {
+    height: 60px;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    margin-right: 150px;
   }
 
-  .contact-section {
-    display: none;
+  .nav-link {
+    list-style: none;
+  }
+
+  .nav-link a {
+    font-family: "Roboto", sans-serif;
+    text-decoration: none;
+    color: white;
+    margin-left: 8px;
+    margin-right: 34px;
+  }
+
+  .envelope {
+    background-image: url("/assets/images/mail.svg");
+    background-repeat: no-repeat;
+    width: 24px;
+    height: 24px;
+  }
+
+  /* Work Section */
+  .grid-items {
+    display: grid;
+    grid-template-rows: repeat(6, 1fr);
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-areas:
+      "heading divider"
+      "main main"
+      "card1 card2"
+      "card3 card4"
+      "card5 card6";
+    justify-content: center;
+  }
+
+  .main-card-content {
+    display: flex;
+    flex-direction: column;
+    height: 201px;
+    justify-content: center;
+  }
+
+  .work-main-heading {
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 30px;
+    line-height: 52px;
+    color: #1c1a19;
+    align-self: center;
+    grid-area: heading;
+  }
+
+  .divider {
+    width: 400px;
+    max-width: calc(100% - 18px);
+    border: 1px solid #212121;
+    grid-area: divider;
+  }
+
+  .main-card {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    grid-area: main;
+  }
+
+  .post-image {
+    background-image: url("/assets/images/post-image.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    margin-right: 20px;
+    align-self: normal;
+    height: 426px;
+  }
+
+  .work-title-post {
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    max-width: 250px;
+    color: #091e42;
+    min-width: 325px;
+    margin-top: 24px;
+    margin-bottom: 15px;
   }
 
-  .footer {
+  .work-supporting-text {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    max-width: 426px;
+  }
+
+  .languages-list {
+    display: flex;
+    width: 322px;
+    margin-top: 23px;
+    margin-bottom: 24px;
+  }
+
+  .languages-list li {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    list-style: none;
+    color: #091e42;
+    border: 1px solid #8993a4;
+    margin-right: 5px;
+    padding: 5px;
+  }
+
+  .card {
     display: none;
   }
+
+  .card-desktop {
+    display: flex;
+    flex-direction: column;
+    margin-top: 10px;
+    margin-bottom: 20px;
+    justify-content: flex-end;
+    align-items: center;
+  }
```

Comment:
- [ ] As for the hover effect on the cards suggested by the previous reviewer, i think it is possible to add this effect with pure CSS. The steps i would follow to achieve this is as follows:

- Increase the opacity of the images of the cards (You can use a filter CSS attribute to add opacity)
- use `display: none` to hide the see project button

When a user hovers a card,

- Use `display: none to hide the description and tags
- Remove the opacity
- use `display: block` to show the See project button. 

This is my thought process so you can have an idea of how to go about it.

Reply:
Hi @vickymarz, Thank you so much you really helped me it's working now. Once again thank you much.